### PR TITLE
Marketing Pages - Modal

### PR DESCRIPTION
**Pull request by andrewjcasal**
## Description

Modal refactored out into .js and .scss
Additional functionality for playing and pausing YouTube videos.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by andrewjcasal** [comment]
Hey @usirin can you review this? 😀

File: docs/_includes/js/modal.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            overlay.classList.remove('isShown');
+            modal.classList.remove('isShown');
+            if (youtube) {
+              toggleVideo(youtube, 'hide');
+            }
+          })
+        });
+      });
+    })
+	}
```
Comment (sinan) [inline docs/_includes/js/modal.js]: what up with the indentation above?

Comment (sinan) [inline docs/_includes/js/modal.js]: this is not a util, therefore `modal.js` name is misleading. you can either call it `store-modal.js` or make it generic and keep the name same.

**Review by sinan — CHANGES_REQUESTED** [review]
thx @andrewjcasal, just minor changes needed 👍

File: docs/_includes/js/modal.js (hunk 1 above)
Comment (andrewjcasal) [inline docs/_includes/js/modal.js]: The modal's being used on the Trials page, and will be used on the docs training video page too. Where would be the best place to put this as a util? And what should it be called then?

**Review by sinan — APPROVED** [review]
👍

**Comment by usirin** [comment]
@sinan left this PR to you, as you know better this area.

**Comment by sinan** [comment]
@andrewjcasal taking this in but please take a note to fix the modal util to be more generic and allowing for multiple modals, and removing video controls from the modal logic.

**Pull request merged**